FLIR/conservator/cli/cvc.py

```diff
@@ -333,6 +336,22 @@ def status(local_dataset):
 @pass_valid_local_dataset
 @check_git_config
 def download(local_dataset, include_raw, include_analytics, pool_size, symlink, tries):
+    if include_raw and not check_dir_access(local_dataset.data_path):
+        click.secho(f"Cannot write to {local_dataset.data_path}!", fg="red", bold=True)
+        sys.exit(1)
+
+    if include_analytics and not check_dir_access(local_dataset.raw_data_path):
+        click.secho(f"Cannot write to {local_dataset.data_path}!", fg="red", bold=True)
+        sys.exit(1)
+
+    if not check_dir_access(local_dataset.cache_path):
+        click.secho(
+            f"Cannot write to cache directory ({local_dataset.cache_path})!",
+            fg="red",
+            bold=True,
+        )
+        sys.exit(1)
```

Comment:
The later call to local_dataset.download() creates the paths passed to the check_dir_access() calls, but check_dir_access() will return False because those directories likely won't exist yet.